src/query/graphite/common/percentiles.go

```diff
@@ -71,6 +71,16 @@ func SafeSort(input []float64) int {
 	return nans
 }
 
+func SafeSum(input []float64) float64 {
```

Comment:
nit: Need comments for exported methods.

Reply:
Added